.travis.yml

```diff
@@ -4,19 +4,26 @@ addons:
   apt:
     packages:
       - libgmp-dev
-      - openjdk-6-jre
+      - oracle-java8-installer
 
 env:
- # - GHCVER=7.6.3  ESVER=1.6.0 # Deprecated
- # - GHCVER=7.8.3  ESVER=1.0.3 # Deprecated
- # - GHCVER=7.8.3  ESVER=1.1.2 # Deprecated
- # - GHCVER=7.8  ESVER=1.2.4 # deprecated
- - GHCVER=7.8  ESVER=1.3.6
- - GHCVER=7.8  ESVER=1.4.1
- - GHCVER=7.10 ESVER=1.5.2
- - GHCVER=7.10 ESVER=1.6.0
- - GHCVER=7.10 ESVER=1.7.2
- - GHCVER=8.0  ESVER=1.7.2
+ global:
+  - JAVA_HOME=/usr/lib/jvm/java-8-oracle
+ matrix:
+   - GHCVER=7.8  ESVER=1.3.6 ESFLAG=ES1 DLINK=https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch
+   - GHCVER=7.8  ESVER=1.4.1 ESFLAG=ES1 DLINK=https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch
+   - GHCVER=7.10 ESVER=1.5.2 ESFLAG=ES1 DLINK=https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch
+   - GHCVER=7.10 ESVER=1.6.0 ESFLAG=ES1 DLINK=https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch
+   - GHCVER=7.10 ESVER=1.7.2 ESFLAG=ES1 DLINK=https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch
+   - GHCVER=8.0  ESVER=1.7.2 ESFLAG=ES1 DLINK=https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch
+   - GHCVER=8.0  ESVER=2.4.2 ESFLAG=ES1 DLINK=https://download.elastic.co/elasticsearch/release/org/elasticsearch/distribution/tar/elasticsearch/2.4.2/elasticsearch
+   - GHCVER=8.0  ESVER=2.4.2 ESFLAG=ES5 DLINK=https://download.elastic.co/elasticsearch/release/org/elasticsearch/distribution/tar/elasticsearch/2.4.2/elasticsearch
+   - GHCVER=8.0  ESVER=5.0.2 ESFLAG=ES5 DLINK=https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch ES_JAVA_OPTS="-Xms500m -Xmx750m"
+
+matrix:
+  allow_failures:
```

Comment:
I'd say sure, or comment them out. I imagine if I'm working on v2 backports I'll create a branch, reenable the tests and work from there.